jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Stream.java

```diff
@@ -1053,4 +1058,9 @@ public List<StreamFrame> getFrames()
             return frames;
         }
     }
+
+    public interface DispatchableListener extends Stream.Listener
+    {
+        void onDataAvailable(Stream stream, boolean dispatch);
```

Comment:
This method should have javadoc, at least to explain the `boolean dispatch` parameter and the the fact that it solves a recursion problem.